#!/usr/bin/env python
PACKAGE = "livox_hikcamera_cal"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("x_max", double_t, 0, "x maximun value", 40, -40, 40)
gen.add("y_max", double_t, 0, "y maximun value", 40, -40, 40)
gen.add("z_max", double_t, 0, "z maximun value", 40, -40, 40)
gen.add("x_min", double_t, 0, "x minimum value", -40, -40, 40)
gen.add("y_min", double_t, 0, "y minimum value", -40, -40, 40)
gen.add("z_min", double_t, 0, "z minimum value", -40, -40, 40)
# gen.add("double_param", double_t, 0, "A double parameter", 1.0, 0.0, 5.0)
# gen.add("str_param", str_t, 0, "A string parameter", "hello")
# gen.add("bool_param", bool_t, 0, "A boolean parameter", True)

exit(gen.generate(PACKAGE, PACKAGE, "PointcloudFilter"))